Twits/Twits/Views/Twit/Create.cshtml

```diff
@@ -0,0 +1,40 @@
+﻿@model Twits.Models.Twit
+
+@{
+    ViewBag.Title = "Create";
+}
+
+<h2>Create</h2>
+
+
+@using (Html.BeginForm())
+{
+    @Html.AntiForgeryToken()
```

Comment:
Don't forget to add the `ValidateAntiForgeryToken` attribute to the POST action method.